docs/pages/versions/unversioned/sdk/url.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: URL
+description: Standard URL API available on all Expo-supported platforms. 
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo'
+packageName: 'expo'
+---
+
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+
+The standard URL API is available on all Expo-supported platforms.
+
+<PlatformsSection android emulator ios simulator web />
+
+## Installation
+
+This API is part of the `expo` package. On native platforms, built-in `URL` and `URLSearchParams` implementations replace the shims in `react-native`. Refer to the [browser and server runtime support](https://caniuse.com/url) for web and Node.js.
+
+## Usage
+
+```js
+const url = new URL('https://expo.dev');
+
+const params = new URLSearchParams();
+```
+
+## Conformance
+
+Expo's build-in URL support attempts to be fully [spec compliant](https://developer.mozilla.org/en-US/docs/Web/API/URL). 
+
+The only missing exception is that native platforms do not currently support [unicode characters](https://unicode.org/reports/tr46/) in the hostname. 
```

Comment:
```suggestion
The only missing exception is that native platforms do not currently support [non-ASCII characters](https://unicode.org/reports/tr46/) in the hostname. 
```
Again, all characters are Unicode characters, it's characters outside of the ASCII U+007F range that need to be converted to punycode